stage0/src/asm/boot.s

```diff
@@ -46,6 +46,18 @@ _protected_mode_start:
     # Set up a basic stack, as we may get interrupts.
     mov $stack_start, %esp
 
+    # Set up the 4K page table for the lowest 2 MiB of memory.
```

Comment:
I think your commit message would also be excellent context here as a comment (especially since you already wrote it), but up to you.